tests/cypress/support/helpers/lending-mocks.ts

```diff
@@ -0,0 +1,82 @@
+import { oneAddress, oneFloat, oneInt, oneOf, onePrice, range } from '@/support/generators'
+import { oneToken } from '@/support/helpers/tokens'
+import type { GetMarketsResponse } from '@curvefi/prices-api/src/llamalend/responses'
+
+const LendingChains = ['ethereum', 'fraxtal', 'arbitrum']
+
+export const mockLendingChains = () =>
+  cy.intercept('https://prices.curve.fi/v1/lending/chains', { body: { data: LendingChains } })
+
+const oneLendingPool = (chain: string, utilization: number): GetMarketsResponse['data'][number] => {
```

Comment:
I wrote one of the mocks as a data generator to notice the difference. I actually think that:
- it's easier to maintain than json files
- it's much more readable
- it contains business logic instead of some blindly copied file
- it allowed me to find some bugs in the API parsers
- it makes the tests more independent on the specific data, making it easier to change the data without breaking tests